sktime/registry/_lookup.py

```diff
@@ -134,13 +134,18 @@ def _is_estimator(name, klass):
     # packages
     with warnings.catch_warnings():
         warnings.simplefilter("ignore", category=FutureWarning)
+        warnings.simplefilter("module", category=ImportWarning)
         for _, module_name, _ in pkgutil.walk_packages(path=[ROOT], prefix="sktime."):
 
             # Filter modules
             if _is_ignored_module(module_name) or _is_private_module(module_name):
                 continue
 
-            module = import_module(module_name)
+            try:
+                module = import_module(module_name)
+            except ModuleNotFoundError as e:
+                warnings.warn(str(e), ImportWarning)
+                continue
```

Comment:
@fkiraly it doesn't matter, but this line is where access to `module` was prevented.  The effect of a `continue` statement is to suddenly jump to the next loop iteration, skipping the rest of the loop body below.

Reply:
ah, I missed that, got it!

Reply:
>@fkiraly it doesn't matter,

yes, you were right. I missed the `continue`, so this is not-an-issue